packages/react-components/react-charts-preview/src/utilities/FocusableTooltipText.tsx

```diff
@@ -1,80 +1,66 @@
+/* eslint-disable no-restricted-globals */
```

Comment:
why added this rule?

Reply:
without this rule it says "window" cannot be used. we have to import it from useFluent() but that was also throwing error